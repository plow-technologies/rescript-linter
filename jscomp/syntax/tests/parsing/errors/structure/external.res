// missing JS value name
external setTimeout: (unit => unit, int) => float =
