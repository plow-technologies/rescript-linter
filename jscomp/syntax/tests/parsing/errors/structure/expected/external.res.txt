
  Syntax error!
  tests/parsing/errors/structure/external.res:2:51

  1 │ // missing JS value name
  2 │ external setTimeout: (unit => unit, int) => float =
  3 │ 

  An external requires the name of the JS value you're referring to, like "setTimeout".

external setTimeout : (unit -> unit) -> int -> float